pkg/quarkus/v1alpha/api.go

```diff
@@ -116,3 +158,83 @@ func (p *createAPISubcommand) InjectResource(res *resource.Resource) error {
 
 	return nil
 }
+
+// findOldFilesForReplacement verifies marker (## marker) and if it found then merge new api CRD file to the odler logic
+func findOldFilesForReplacement(path, newfile string) bool {
+
+	f, err := os.Open(path)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// remember to close the file at the end of the program
+	defer f.Close()
+
+	// read the file line by line using scanner
+	scanner := bufio.NewScanner(f)
+	var foundMarker bool
+	for scanner.Scan() {
+		// do something with a line
+		if scanner.Text() == "## marker" {
+			foundMarker = true
+			break
+		}
+	}
+
+	if foundMarker {
+		scanner.Scan()
+		catLine := scanner.Text()
+
+		splitByPipe := strings.Split(catLine, "|")
+
+		finalString := strings.TrimSuffix(strings.TrimPrefix(strings.TrimSpace(splitByPipe[0]), "cat"), "target/kubernetes/kubernetes.yml")
+
+		updatedLine := "	" + "cat" + finalString + newfile + " target/kubernetes/kubernetes.yml" + " |" + splitByPipe[1]
+		// fmt.Printf("merge : %s\n", merge)
+
+		if err := scanner.Err(); err != nil {
+			log.Fatal(err, "Failed to scan the line from Makefile")
```

Comment:
That's a good point, would a log level of `Error` be better than `Fatal`? I don't see anywhere after logging with the fatal level that the operation is being aborted.

Reply:
Or we could make it so that the command does not finish executing and rather exits with a non-zero exit code after the fatal log

Reply:
Modified like this.

`			log.Error(err, "Unable to scan existing bundle target command from the Makefile. New bundle target command being created. This may overwrite any existing commands.")
`